sass/index.scss

```diff
@@ -1,5 +1,3 @@
-@import "eyeglass/assets";
```

Comment:
Details regarding why are in [my comment below](https://github.com/sass-eyeglass/eyeglass-spriting/pull/19#issuecomment-167058841).